backend/src/db/db.client.ts

```diff
@@ -11,14 +12,24 @@ const dbOptions: dbOptionsType = {
   useNewUrlParser: true,
 };
 
-const client: any = new MongoClient(dbURI, dbOptions);
+let db: Db;
+const client: MongoClient = new MongoClient(dbURI, dbOptions);
 
-export = async (): Promise<void> => {
+export const mongoConnect =  async (): Promise<void> => {
```

Comment:
jeśli przetestowałeś połączenie z bazą i wszystko jest w porządku, to nie ma się do czego przyczepić

Reply:
baza wstaje